lib/jxl/modular/encoding/encoding.cc

```diff
@@ -444,8 +444,25 @@ Status ModularDecode(BitReader *br, Image &image, GroupHeader &header,
   const ANSCode *code = &code_storage;
   const std::vector<uint8_t> *context_map = &context_map_storage;
   if (!header.use_global_tree) {
-    size_t tree_size_limit = 1024 + image.w * image.h * nb_channels;
-    JXL_RETURN_IF_ERROR(DecodeTree(br, &tree_storage, tree_size_limit));
+    size_t total_pixels = 1024;
```

Comment:
Maybe nicer to call this variable `max_tree_size`? It might be slightly confusing this way.

Reply:
Sure, one sec.